target/linux/octeon/image/edgerouter_gen_usb_img.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -x
+[ $# -eq 5 ] || {
+    echo "SYNTAX: $0 <file> <bootfs image> <rootfs image> <bootfs size> <rootfs size>"
+    exit 1
+}
+
+OUTPUT="$1"
+BOOTFS="$2"
+ROOTFS="$3"
+BOOTFSSIZE="$4"
+ROOTFSSIZE="$5"
+
+head=4
+sect=63
+
+set `ptgen -o $OUTPUT -h $head -s $sect -l 1024 -t b -p ${BOOTFSSIZE}M -t 83 -p ${ROOTFSSIZE}M`
```

Comment:
The backticks should be replaced by preferred $() syntax.

Reply:
Refreshed script to match ```/linux/sunxi/image/gen_sunxi_sdcard_img.sh```